CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## Changed
```

Comment:
Missing the version number here.

Reply:
This is deliberate, can't release until Jenkins is updated.